README.md

```diff
@@ -0,0 +1,25 @@
+# Server Duels
+
+## Description
+
+Server Duels is a discord bot that allows you to duel your friends in turn-based combat.
+
+## Table of Contents
+
+- [Features](#features)
+- [Contributing](#contributing)
+- [License](#License)
+
+## Features
+Server Duels plans to feature turn-based text combat between members of a discord server. Over
+time be will be adding more features and updating the bot.
+
+## Contributing
+Contributors are welcome! Please follow the instructions at these two links to contribute to
+the bot.
+- [Setting Up Your Development Environment](https://github.com/domhankle/server-duels/wiki/Setting-Up-Your-Development-Environment)
+- [Contributing to Server Duels](https://github.com/domhankle/server-duels/wiki/Contributing-to-Server-Duels)
+
+## License
+For license information, see the LICENSE.md file included in the repo.
```

Comment:
I think we can remove this "License" section since we will provide a link straight to it in the "Table of Contents" section.